helm/charts/determined/templates/genai/genai-queue-service.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.genai }}
+{{- if .Values.genai.version }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: genai-queue-service-{{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: genai-{{ .Release.Name }}
+    release: {{ .Release.Name }}
+spec:
+  ports:
+  - port: {{ required "A valid Values.genai.messageQueuePort entry required!" .Values.genai.messageQueuePort }}
```

Comment:
that's a new field, right? maybe values.yaml needs an update with a commented out example.

Reply:
It is, we actually removed the examples from the values.yaml when I was talking with @NicholasBlaskey a bit ago: https://github.com/determined-ai/determined/pull/8821/files

The idea being that the values would be confusing to new users and that genai isn't fully released yet.

Instead we have documentation here: https://hpe-ai-solutions-documentation.netlify.app/products/gen-ai/latest/admin/set-up/install-kubernetes/ which I will update to include that parameter